examples/tutorials/nb_python/asset_viewer.py

```diff
@@ -0,0 +1,972 @@
+# and values for the passed UV-Sphere primitive template. The values are tuples with
+# the first entry being the value,the second being whether the property is
+# editable and the third being the type.
+def build_dict_of_UVSphere_prim_attrs(uvsphere_template):
+    res_dict = build_dict_of_prim_attrs(uvsphere_template)
+    return res_dict
+
+
+# This method will deduce the appropriate template type and build the subsequent
+# dictionary containing containing k-v pairs of template property names
+# and values for the passed template. The values are tuples with
+# the first entry being the value,the second being whether the property is
+# editable and the third being the type.
+def build_dict_from_template(template):
+    template_class = template.template_class
+    if "ObjectAttributes" in template_class:
+        return build_dict_of_Object_attrs(template)
+    if "StageAttributes" in template_class:
+        return build_dict_of_Stage_attrs(template)
+    if "PhysicsManagerAttributes" in template_class:
+        return build_dict_of_PhysicsSim_attrs(template)
+    if "CapsulePrimitiveAttributes" in template_class:
+        return build_dict_of_Capsule_prim_attrs(template)
+    if "ConePrimitiveAttributes" in template_class:
+        return build_dict_of_Cone_prim_attrs(template)
+    if "CubePrimitiveAttributes" in template_class:
+        return build_dict_of_Cube_prim_attrs(template)
+    if "CylinderPrimitiveAttributes" in template_class:
+        return build_dict_of_Cylinder_prim_attrs(template)
+    if "IcospherePrimitiveAttributes" in template_class:
+        return build_dict_of_Icosphere_prim_attrs(template)
+    if "UVSpherePrimitiveAttributes" in template_class:
+        return build_dict_of_UVSphere_prim_attrs(template)
+    print("Unknown template type : %s " % template_class)
+    return None
+
+
+# This will set a template's attributes from the passed dictionary
+def set_template_properties_from_dict(template, template_dict):
+    for k, v in template_dict.items():
+        setattr(template, k, v[0])
+    return template
+
+
+# This will display all the properties of an attributes template
+def show_template_properties(template):
+    template_dict = build_dict_from_template(template)
+    print("Template {} has : ".format(template.handle))
+    for k, v in template_dict.items():
+        print(
+            "\tProperty {} has value {} of type {} that is editable : {}".format(
+                k, v[0], v[2], v[1]
+            )
+        )
+
+
+# %%
+# @title Define Configuration Utility Functions { display-mode: "form" }
+# @markdown (double click to show code)
+
+# @markdown This cell defines a number of utility functions used throughout the tutorial to make simulator reconstruction easy:
+# @markdown - make_cfg
+# @markdown - make_default_settings
+# @markdown - make_simulator_from_settings
+
+
+def make_cfg(settings):
+    sim_cfg = habitat_sim.SimulatorConfiguration()
```

Comment:
Why can't we use the utility we made for this?

Reply:
I cannibalized an existing ECCV colab to get started here.  These all need to be updated.